mypy_protobuf/main.py

```diff
@@ -141,6 +142,29 @@ def _add_messages(
             _add_enums(fd.enum_type, start_prefix, fd)
 
 
+class GRPCType(enum.Enum):
+    SYNC = "SYNC"
+    ASYNC = "ASYNC"
+    BOTH = "BOTH"
+
+    @classmethod
+    def from_parameter(cls, parameter: str) -> GRPCType:
```

Comment:
handle error case if someone passes in both options, either error with invalid option, or set both. I'd lean towards error

Reply:
done